docs/src/documentation-pages/dev/pe.md

```diff
@@ -0,0 +1,42 @@
+# P&E database setup instructions
+
+## Local
+
+To create the P&E database locally, first run Crossfeed with `npm start`. Then run:
+
+```
+cd backend
+npm run pesyncdb
+```
+
+## Accessor
+
+### Add credentials to SSM
+
+Before deploying. Generate a secure secret value for a database password, then run the following commands on the terraformer instance:
+
+```
+aws ssm put-parameter --name "/crossfeed/prod/PE_DATABASE_NAME" --value "pe" --type "SecureString"
```

Comment:
Remember that once you put these parameters, the current accessor instance you're using is on Crossfeed staging -- so you'd need to specify `/crossfeed/staging/PE_DATABASE_NAME`.

Once we get the whole flow working, we can move it to prod.